BridgeSDK/BridgeAPI/SBBUploadManager.m

```diff
@@ -430,18 +430,24 @@ - (void)URLSession:(NSURLSession *)session task:(NSURLSessionTask *)task didComp
     NSMutableDictionary *headers = [NSMutableDictionary dictionary];
     [self.authManager addAuthHeaderToHeaders:headers];
     [self.networkManager post:ref headers:headers parameters:nil completion:^(NSURLSessionDataTask *task, id responseObject, NSError *error) {
+
 #if DEBUG
-      if (error) {
-        NSLog(@"Error calling upload complete for upload ID %@:\n%@", uploadSession.id, error);
-      } else {
-        NSString* uploadStatusUrlString = nil;
-        if ([self.networkManager isKindOfClass:[SBBNetworkManager class]]) {
-          NSString* relativeUrl = [NSString stringWithFormat:@"/api/v1/upload/%@/status", uploadSession.id];
-          NSURL* url = [(SBBNetworkManager*) self.networkManager URLForRelativeorAbsoluteURLString:relativeUrl];
-          uploadStatusUrlString = [url absoluteString];
+        if (error) {
+            NSLog(@"Error calling upload complete for upload ID %@:\n%@", uploadSession.id, error);
+        } else {
+            NSString* uploadStatusUrlString = nil;
+            if ([self.networkManager isKindOfClass:[SBBNetworkManager class]]) {
+                NSString* relativeUrl = [NSString stringWithFormat:@"/api/v1/upload/%@/status", uploadSession.id];
+                NSURL* url = [(SBBNetworkManager*) self.networkManager URLForRelativeorAbsoluteURLString:relativeUrl];
+                uploadStatusUrlString = [url absoluteString];
+                
+                if ([_uploadDelegate respondsToSelector:@selector(uploadManager:uploadOfFile:completedWithVerificationURL:)]) {
+                    
```

Comment:
(FYI if your Xcode editor is set to 4-space tabs, just select-all/cut/paste and it will fix all the tabs in the file)